app/api/authentication/helpers.rb

```diff
@@ -0,0 +1,34 @@
+module Authentication
+  module Helpers
+    extend ActiveSupport::Concern
+
+    included do
+      use Middleware::JwtMiddleware
+      rescue_from JWT::DecodeError do |_e|
```

Comment:
Does the `_e` convention mean "unused argument `e`"?

Reply:
Yeah, totally - sometimes you even just do this:

```ruby
rescue_from JWT::DecodeError do |_|
```

Reply:
I would usually just use `_` as @jonallured points out. That's also convention in some other languages (eg Scala).